datasets/hate_offensive/README.md

```diff
@@ -0,0 +1,149 @@
+---
+
+YAML tags:
+annotations_creators:
+- crowdsourced
+language_creators:
+- machine-generated
+languages:
+- en
+licenses:
+- mit
+multilinguality:
+- monolingual
+size_categories:
+- n<1K
+source_datasets:
+- original
+task_categories:
+- text-classification
+task_ids:
+- multi-class-classification
+---
+
+# Dataset Card for HateOffensive
+
+## Table of Contents
+- [Dataset Description](#dataset-description)
+  - [Dataset Summary](#dataset-summary)
+  - [Supported Tasks](#supported-tasks-and-leaderboards)
+  - [Languages](#languages)
+- [Dataset Structure](#dataset-structure)
+  - [Data Instances](#data-instances)
+  - [Data Fields](#data-instances)
+  - [Data Splits](#data-instances)
+- [Dataset Creation](#dataset-creation)
+  - [Curation Rationale](#curation-rationale)
+  - [Source Data](#source-data)
+  - [Annotations](#annotations)
+  - [Personal and Sensitive Information](#personal-and-sensitive-information)
+- [Considerations for Using the Data](#considerations-for-using-the-data)
+  - [Social Impact of Dataset](#social-impact-of-dataset)
+  - [Discussion of Biases](#discussion-of-biases)
+  - [Other Known Limitations](#other-known-limitations)
+- [Additional Information](#additional-information)
+  - [Dataset Curators](#dataset-curators)
+  - [Licensing Information](#licensing-information)
+  - [Citation Information](#citation-information)
+
+## Dataset Description
+- **Homepage** : https://arxiv.org/abs/1905.12516 
+- **Repository** : https://github.com/t-davidson/hate-speech-and-offensive-language
+- **Paper** : https://arxiv.org/abs/1905.12516 
+- **Leaderboard** : 
+- **Point of Contact** : trd54 at cornell dot edu
+
+### Dataset Summary
+
+### Supported Tasks and Leaderboards
+[More Information Needed]
+
+### Languages
+English (`en`)
+
+## Dataset Structure
+
+### Data Instances
+{
+"count": "3",
+ "hate_speech": "0",
+ "offensive_language": "0",
+ "neither": "3",
+ "class": "2",
+ "tweet": "!!! RT @mayasolovely: As a woman you shouldn't complain about cleaning up your house. &amp; as a man you should always take the trash out...")
+}
```

Comment:
Nice, could you wrap this in a code block? (means adding three backlashes ` before and after)
Also, the names of the fields do not match the ones you specified in the loading script